### PR TITLE
feat: credits component

### PR DESCRIPTION
**Pull request by cindy-sf**

### CODE REVIEW

File: src/components/Credits/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const CreditsWrapper = styled.View`
+  padding: ${spaces.SMALL}px;
+  margin-top: ${spaces.XXX_LARGE}px;
+`
+
+const Credits = () => (
+  <CreditsWrapper>
+    <Text size="BODY_2" font="POPPINS_ITALIC">
+      Powered by St Fleurant Team
```
Comment (j0nathansf) [inline src/components/Credits/index.tsx]: I would have preferred SFTeam but no worries :)

**Review by j0nathansf — DISMISSED** [review]
Good job

File: src/components/Credits/index.tsx (hunk 1 above)
Comment (cindy-sf) [reply src/components/Credits/index.tsx]: Ok 👍

Comment (cindy-sf) [reply src/components/Credits/index.tsx]: Done ✔️

**Review by j0nathansf — APPROVED** [review]

**Pull request merged**